dpctl-capi/tests/test_sycl_queue_interface.cpp

```diff
@@ -41,20 +41,31 @@ namespace
 {
 DEFINE_SIMPLE_CONVERSION_FUNCTIONS(queue, DPCTLSyclQueueRef);
 
+void error_handler_fn(int err)
```

Comment:
Do not suppress the warning in this way. Instead use,
```
void error_handler_fn(int /**/)
```

Reply:
Done.